tests/test_pycdf_istp.py

```diff
@@ -523,6 +524,14 @@ def testValidScale(self):
              'SCALEMIN type CDF_INT2 does not match variable type CDF_BYTE.'
              ],
              errs)
+        tt2000.attrs['SCALEMIN'] = 200.42e45 #tt2000 testing
```

Comment:
probably better to use tt2000.attrs.new() so you can specify the type explicitly instead of having to use a very very large number.